Client/Frontend/Popup/PaymentHandlerPopupView.swift

```diff
@@ -0,0 +1,284 @@
+        titleLabel.textAlignment = .center
+        titleLabel.font = UIFont.systemFont(ofSize: titleSize, weight: titleWeight)
+        titleLabel.text = title
+        titleLabel.numberOfLines = 0
+        containerView.addSubview(titleLabel)
+        
+        messageLabel = UILabel(frame: CGRect.zero)
+        messageLabel.textColor = BraveUX.greyH
+        messageLabel.textAlignment = .center
+        messageLabel.font = UIFont.systemFont(ofSize: 15, weight: UIFont.Weight.regular)
+        messageLabel.text = message
+        messageLabel.numberOfLines = 0
+        containerView.addSubview(messageLabel)
+        
+        displayItemTitle = UILabel(frame: CGRect.zero)
+        displayItemTitle.textColor = BraveUX.greyH
+        displayItemTitle.textAlignment = .left
+        displayItemTitle.font = UIFont.systemFont(ofSize: 13, weight: UIFont.Weight.bold)
+        displayItemTitle.text = Strings.orderSummary
+        displayItemTitle.numberOfLines = 0
+        containerView.addSubview(displayItemTitle)
+        
+        totalItemTitle = UILabel(frame: CGRect.zero)
+        totalItemTitle.textColor = BraveUX.greyH
+        totalItemTitle.textAlignment = .left
+        totalItemTitle.font = UIFont.systemFont(ofSize: 13, weight: UIFont.Weight.bold)
+        totalItemTitle.text = Strings.total
+        totalItemTitle.numberOfLines = 0
+        containerView.addSubview(totalItemTitle)
+        
+        addTotalLabel(message: "")
+        
+        if let inputType = inputType {
+            textField = UITextField(frame: CGRect.zero).then {
+                $0.keyboardType = inputType
+                $0.textColor = .black
+                $0.placeholder = inputPlaceholder ?? ""
+                $0.autocorrectionType = .no
+                $0.autocapitalizationType = .none
+                $0.layer.cornerRadius = 4
+                $0.layer.borderColor = UIColor(white: 0, alpha: 0.3).cgColor
+                $0.layer.borderWidth = 1
+                $0.delegate = self
+                $0.textAlignment = .center
+                $0.isSecureTextEntry = secureInput
+                containerView.addSubview($0)
+            }
+        }
+        
+        updateSubviews()
+        
+        setPopupContentView(view: containerView)
+        setStyle(popupStyle: .dialog)
+        setDialogColor(color: BraveUX.popupDialogColorLight)
+    }
+            
+    func addDisplayItemLabel(message: String) {
+        let displayItem = UILabel(frame: CGRect.zero)
+        displayItem.textColor = BraveUX.greyH
+        displayItem.textAlignment = .left
+        displayItem.font = UIFont.systemFont(ofSize: 12, weight: UIFont.Weight.regular)
+        displayItem.text = message
+        displayItem.numberOfLines = 0
+        
+        containerView.addSubview(displayItem)
+        displayItems.append(displayItem)
+        
+        updateSubviews()
+    }
+    
+    func addTotalLabel(message: String) {
+        totalItem = UILabel(frame: CGRect.zero)
```

Comment:
No need for `CGRect.zero` you can just use `.zero`

Reply:
Don't even need that:

```suggestion
        totalItem = UILabel()
```